articles/hooks/guides/create-delete-hooks-using-cli.md

```diff
@@ -6,20 +6,28 @@ topics:
     - cli
 contentType: how-to
 useCase: extensibility-hooks
+v2: true
 ---
+# Create/Delete Hooks Using the Command-Line Interface
```

Comment:
Rachel and I have been putting new how-to docs in a /guides subfolder to keep them organized.